vendor.mod

```diff
@@ -145,18 +153,18 @@ require (
 	github.com/containerd/console v1.0.4 // indirect
 	github.com/containerd/go-cni v1.1.10 // indirect
 	github.com/containerd/go-runc v1.1.0 // indirect
-	github.com/containerd/nydus-snapshotter v0.14.0 // indirect
-	github.com/containerd/stargz-snapshotter/estargz v0.15.1 // indirect
-	github.com/containerd/ttrpc v1.2.5 // indirect
-	github.com/containernetworking/cni v1.2.2 // indirect
+	github.com/containerd/nydus-snapshotter v0.14.1-0.20241016012921-e55ae70fd45a // indirect
```

Comment:
Looks like they tagged v0.15.0, so we can use a released version; https://github.com/containerd/nydus-snapshotter/releases/tag/v0.15.0

Reply:
Pulled in